tutorial.md

```diff
@@ -1,101 +1,85 @@
 ---
-title: Tutorial - Profile a Vulkan Application on Android
+title: Getting Started
 layout: default
-permalink: /docs/tutorial
+permalink: /docs/getting-started
 ---
 
-> (Short walkthrough: download, install, capture system trace, capture API
-> trace, replay, show counters)
-
 ## Install adb
 
-Make sure the `adb` tool is installed on your host computer. See the
-[adb documentation](https://developer.android.com/studio/command-line/adb) for
-installation instructions.
+Make sure the `adb` tool is installed on your host computer. See the [adb documentation](https://developer.android.com/studio/command-line/adb) for installation instructions.
+
+## Prepare your device
+
+For the duration of the developer preview, instructions on how to set up the beta GPU drivers will be e-mailed to you.
 
-## Profile a Vulkan application
+## Prepare your application
 
-Connect one of the
-[supported devices](../requirements#supported-android-devices)
-to your host computer with a USB cable. Your device must have
-[`adb` debugging enabled](https://developer.android.com/studio/command-line/adb#Enabling)
+* The target application must be debuggable; the [debuggable attribute] in the Android manifest of the application must be set to `true`. This attribute enables AGI to add its own Vulkan layer when the application starts.
+  ```xml
+  <application [...] android:debuggable="true">
+  ```
+[debuggable attribute]: https://developer.android.com/guide/topics/manifest/application-element#debug
+
+## Capturing a systems profile of a Vulkan application
+
+1. Connect one of the [supported devices](../requirements#supported-android-devices) to your host computer with a USB cable. Your device must have [`adb` debugging enabled](https://developer.android.com/studio/command-line/adb#Enabling)
```

Comment:
This should just be `(requirements#supported-android-devices)` since you are also moving the requirements page in this PR.

Reply:
See https://github.com/google/agi/pull/182/commits/62bf1b179b61c5e8ae25464df3961aa218ff1bd8